<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>produit</class>
 <widget class="QDialog" name="produit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Lato Black&quot;;
	color: rgb(213, 213, 229);</string>
   </property>
   <property name="text">
    <string>id_produit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Lato Black&quot;;
	color: rgb(213, 213, 229);</string>
   </property>
   <property name="text">
    <string>date_limite</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>260</y>
     <width>491</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 179, 193);
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>600</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/interface/yas.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>610</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 87 15pt &quot;Lato Black&quot;;
	color: rgb(213, 213, 229);
background-color:rgb(3, 38, 77);
border-radius:8px;
}
QPushButton::hover{
background-color:rgb(179, 179, 193);
}</string>
   </property>
   <property name="text">
    <string>supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tri">
   <property name="geometry">
    <rect>
     <x>984</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 87 15pt &quot;Lato Black&quot;;
	color: rgb(213, 213, 229);
background-color:rgb(3, 38, 77);
border-radius:8px;
}
QPushButton::hover{
background-color:rgb(179, 179, 193);
}</string>
   </property>
   <property name="text">
    <string>tri</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mail">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 87 15pt &quot;Lato Black&quot;;
	color: rgb(213, 213, 229);
background-color:rgb(3, 38, 77);
border-radius:8px;
}
QPushButton::hover{
background-color:rgb(179, 179, 193);
}</string>
   </property>
   <property name="text">
    <string>mail</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/interface/beyaaa.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/new/prefix1/gereproduit.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_retour">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>732</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>299</y>
     <width>182</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*background-color: transparent;
color: rgb(255, 255, 255);*/
QLineEdit { 
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
background-color:rgb(179, 179, 193);
border-style: outset;
	color: rgb(0, 0, 0);
border-radius: 10px;
border-color: blue;
min-width: 10em;
padding: 6px;
}
QLineEdit:hover:!pressed {
background-color:rgb(213, 213, 229);
border-style: inset;
background-color:rgb(213, 213, 229);
background-color:rgb(213, 213, 229);
}
/*QLineEdit:selected {
background-color:rgb(179, 179, 193);
border-style: inset;
}*/
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>361</y>
     <width>182</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*background-color: transparent;
color: rgb(255, 255, 255);*/
QLineEdit { 
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
background-color:rgb(179, 179, 193);
border-style: outset;
	color: rgb(0, 0, 0);
border-radius: 10px;
border-color: blue;
min-width: 10em;
padding: 6px;
}
QLineEdit:hover:!pressed {
background-color:rgb(213, 213, 229);
border-style: inset;
background-color:rgb(213, 213, 229);
background-color:rgb(213, 213, 229);
}
/*QLineEdit:selected {
background-color:rgb(179, 179, 193);
border-style: inset;
}*/
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>710</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>tableView_2</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_mail</zorder>
  <zorder>pushButton_retour</zorder>
  <zorder>pushButton_tri</zorder>
  <zorder>dateEdit</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
